src/store/callZome.js

```diff
@@ -1,5 +1,16 @@
 import { isHoloHosted, log } from '@/utils'
 import { logZomeCall, actionType, UndefinedClientError } from '@/store/utils'
+import { RECONNECT_SECONDS } from '@/consts'
+import wait from 'waait'
+
+let undefinedClientCount = 0
+const signalHoloDisconnect = async (state, dispatch) => {
+  dispatch('holochain/resetConnectionState', null, { root: true })
+  // give time for reconnect (convert to ms)
+  await wait(RECONNECT_SECONDS * 1000)
+  if (state.holoClient) return
+  else return dispatch('holochain/signalHoloDisconnect', null, { root: true })
```

Comment:
```suggestion
  if (!state.holoClient) {
    dispatch('holochain/signalHoloDisconnect', null, { root: true })
  }
```

Removed the return, given we don't care about the return value of this function, I think it's confusing to return a value.